src/Ledger/Conway/Conformance/Certs.agda

```diff
@@ -13,7 +13,7 @@ open import Tactic.Derive.DecEq
 open import Ledger.GovernanceActions gs
 private module Certs = Ledger.Certs gs
 open Certs public
-  hiding (DState; ⟦_,_,_⟧ᵈ; GState; ⟦_,_⟧ᵛ; CertState;
+  hiding (DState; GState; CertState;
```

Comment:
```suggestion
  hiding (DState; GState; CertState;
          To-CertEnv; To-DState; To-PState; To-GState; To-CertState; To-DelegEnv;

```
I think you could hide the `To-` constructor instances from `Ledger.Certs` and then instantiate new ones in this conformance version `Certs`, then you can remove all superscripts on constructors in this module as well.  I've made the suggestions below, but if there's a reason you didn't do this already, you can ignore the suggestions.  In that case, maybe say why you think we shouldn't remove superscripts in the conformance version of `Certs`.